### PR TITLE
Pattern and MaxLength support for Input and TextArea

### PR DESCRIPTION
**Pull request by arshx86**
I've added `maxlength` and `pattern` support which is basically regex filter for those components, because [PrimeVue's Key Filter](https://primevue.org/keyfilter/) had it, why ui-thing wouldn't too.

### CODE REVIEW

File: app/components/Ui/Input.vue [hunk 1]
```diff
@@ -13,14 +19,40 @@
       disabled?: boolean;
       required?: boolean;
       type?: string;
-      modelValue?: any;
+      modelValue?: string;
```
Comment (BayBreezy) [inline app/components/Ui/Input.vue:22]: I don't think we should restrict this to string. What about numbers?

Comment (arshx86) [reply app/components/Ui/Input.vue:22]: Ah, yeah my bad.

Comment (BayBreezy) [reply app/components/Ui/Input.vue:22]: Gonna merge it and then add some examples before.
Thank you btw :)

File: app/components/Ui/Input.vue [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+  <input
+    v-bind="props"
+    :value="modelValue"
+    @input="handleInput"
+    :class="styles({ class: props.class })"
+  />
 </template>
 
 <script lang="ts" setup>
+  import { computed } from "vue";
```
Comment (BayBreezy) [inline app/components/Ui/Input.vue:11]: Removing this as it is not needed in nuxt

Comment (BayBreezy) [reply app/components/Ui/Input.vue]: Thanks again @arshx86 

Appreciate it. The cli will be updated with the changes soon.

**Pull request merged**